app/js/lib/foxbox.js

```diff
@@ -295,15 +297,12 @@ export default class Foxbox extends Service {
    * @return {Promise}
    */
   getServiceState(id) {
-    return new Promise((resolve, reject) => {
-      fetchJSON(`${this.origin}/services/${id}/state`)
-        .then(res => {
-          if (!res) {
-            return reject(new Error(`The action couldn't be performed.`));
-          }
+    return fetchJSON(`${this.origin}/services/${id}/state`).then(res => {
```

Comment:
We want to reject the promise if the response is not OK, rather than resolving data that the app can't process.